src/main/java/uk/gov/hmcts/reform/iacaseapi/domain/handlers/presubmit/ReviewHearingRequirementsPreparer.java

```diff
@@ -49,7 +49,13 @@ public boolean canHandle(
 
         final YesOrNo reviewedHearingRequirements =
             asylumCase.read(AsylumCaseFieldDefinition.REVIEWED_HEARING_REQUIREMENTS, YesOrNo.class)
-            .orElseThrow(() -> new IllegalStateException("reviewHearingRequirements flag should be present"));
+                .orElse(null);
+
+        if (reviewedHearingRequirements == null) {
+            final PreSubmitCallbackResponse<AsylumCase> asylumCasePreSubmitCallbackResponse = new PreSubmitCallbackResponse<>(asylumCase);
+            asylumCasePreSubmitCallbackResponse.addError("This is an old case. It doesn't need hearing requirements to be reviewed.");
```

Comment:
As we discussed, we probably need this text reviewed by a content designer and updated.

Reply:
Text reviewed.